src/diffusers/pipelines/__init__.py

```diff
@@ -21,6 +21,7 @@
     from .dit import DiTPipeline
     from .latent_diffusion import LDMSuperResolutionPipeline
     from .latent_diffusion_uncond import LDMPipeline
+    from .multidiffusion_panorama import MultiDiffusionText2PanoramaPipeline
```

Comment:
I think the pipeline can go in `stable_diffusion` directory. wdyt @patrickvonplaten